latest/index.bs

```diff
@@ -375,17 +379,24 @@ custom attributes of the plate group under the `plate` key.
   <dt><strong>rows</strong></dt>
   <dd>A list of JSON objects defining the rows of the plate. Each row object
       defines the properties of the row at the index of the object in the
-      list. If not empty, it MUST contain a `name` key specifying the row
-      name.</dd>
+      list. Each row in the physical plate MUST be defined, even if no wells
+      in the row are defined. Each defined row MUST contain a `name` key
+      specifying the row name.</dd>
   <dt><strong>version</strong></dt>
   <dd>A string defining the version of the specification.</dd>
   <dt><strong>wells</strong></dt>
   <dd>A list of JSON objects defining the wells of the plate. Each well object 
-      MUST contain a `path` key identifying the path to the well subgroup.</dd>
+      MUST contain a `path` key identifying the path to the well subgroup.
+      The `path` MUST consist of a `name` in the `rows` list, a file separator (`/`),
+      and a `name` from the `columns` list, in that order. The `path` MUST NOT contain
+      additional leading or trailing directories.
+      Each well object MUST contain both a `row_index` key identifying the index into
+      the `rows` list and a `column_index` key indentifying the index into
+      the `columns` list. `row_index` and `column_index` MUST be 0-based.</dd>
```

Comment:
I realise that https://github.com/ome/ngff/pull/70 has been added after this PR was opened, but the decision there means these new attributes should now be named `rowIndex` and `columnIndex`.

Reply:
Should be fixed in 7c2536a.